canonical-web-and-design/snapcraft.io
# Fixes and updates around `u-fixed-width`

Fixes #2176

Various fixes around `u-fixed-width` and new grid.
Most notably:
- adding u-clearfix then floats are used in `u-fixed-width`
- removing nested padding from caused by rows in u-fixed-width 

### QA
- ./run or demo
- click through some pages: home, store, search, snap details
- check if everything is in place
- go to publisher pages, obvious ones (snaps, listing, account, etc) and less obvious (name register, name dispute)
- on home page click on first snap flow language selector, check if spacing is correct (expanded panel should take whole width of the page and not have any nested paddings)
